### DISCUSSION TITLE
Black screen issue

### DISCUSSION THREAD

**Discussion by carbinae** [Whisky-App/Whisky#968] [General]
With the new 2.3.2 version, when I open the game nothing appears but a black screen. It all happened after the update. The game I mentioned is Tekken 8, is there a way to fix it?

**Comment by ojh050118** [comment]
I'm having the same problem with version 2.3.2. What was the last version you were able to play?

**Reply by carbinae** [reply]
Must be 2.3.0 or 2.3.1

**Reply by carbinae** [reply]
I tried reinstall 2.3.0 and 2.3.1 and I got black screen again, I’m going to reinstall Tekken, maybe the problem was about games’ folders. I will notify you.

**Reply by carbinae** [reply]
Now I tried EVEN 2.2.4 and am still getting that black screen, I don't know what caused this. I hope they fix it with a new update.

**Reply by ojh050118** [reply]
Could the recent update to Tekken 8 be the problem?

**Reply by carbinae** [reply]
It could be..

**Comment by carbinae** [comment]
I deleted gptk and reinstalled it under 2.2.4 version. It worked.

**Reply by haider0072** [reply]
How to do this can you please tell, do I have to download whisky version or gptk ?

**Reply by carbinae** [reply]
> How to do this can you please tell, do I have to download whisky version or gptk ?
> 
> 

be sure that you deleted all the latest version files of whisky in the library then fresh install 2.2.4
